CNAME

```diff
@@ -1 +1 @@
-sefglobal.org
```

Comment:
Did you make any change in the cname?

Reply:
No, I also host it on a github page. So it changed to auto. I reset it. I hosted a separate branch in my repostery.

Reply:
> https://sef.tute.lk/ What is this? @mkskaushalya

It is the page I used to test.

Reply:
> No, I also host it on a github page. So it changed to auto. I reset it. I hosted a separate branch in my repostery.

Actually we shouldn't do that noh? Because we have published it in our main domain we shouldn't not have another domain so can you remove it from your repository also. Is there a reason for hosting it in your repository? @mkskaushalya

Reply:
> > No, I also host it on a github page. So it changed to auto. I reset it. I hosted a separate branch in my repostery.
> 
> Actually we shouldn't do that noh? Because we have published it in our main domain we shouldn't not have another domain so can you remove it from your repository also. Is there a reason for hosting it in your repository? @mkskaushalya

Definitely. I will remove it.

Reply:
Removed

Reply:
Reason is because you can easily see what I have changed

Reply:
> Reason is because you can easily see what I have changed

Ah for that we have the preview link noh? @mkskaushalya

Reply:
Can you update the preview link? @mkskaushalya 
You should include the pr number except this {PR_NUMBER}
Ex: https://pr-0000-sef-site.surge.sh/

Reply:
Ah Okay

Reply:
I updated preview link